js/src/View/AppView.js

```diff
@@ -0,0 +1,87 @@
+var Backbone = require('backbone');
+var $ = require('jquery');
+var _ = require('underscore');
+var handlebars = require('handlebars');
+
+var UserModel = require('../Model/UserModel.js');
+var appViewTemplate = require('../templates/container.hbs');
+var TodoListView = require('./TodoListView.js');
+var TodoListCollection = require('../Collection/TodoListCollection.js');
+var TodoListModel = require('../Model/TodoListModel.js');
+var todoAppTemplate = require('../templates/todoapp.hbs');
+
+var AppView = Backbone.View.extend({
+    el: 'body',
+    events: {
+        'click .addlist': 'addNewList',
+    },
+
+    initialize: function(){
+
+        this.render();
+
+        this.user = new UserModel();
+        this.user.fetch({
+            success: _.bind(function(model){
+                console.log('Success: user loaded.');
```

Comment:
We can get rid of these console logs once we verify the code path is working and have unit tests for the code path. Let's get rid of them for now, and we can setup a unit test framework this week.